appscale/tools/local_state.py

```diff
@@ -217,6 +217,12 @@ def generate_deployment_params(cls, options, node_layout, additional_creds):
     }
     creds.update(additional_creds)
 
+    if options.update:
+      update_dir_creds = {
+        'update': str(options.update)
```

Comment:
Since this is all being serialized to JSON later, you can just use the list itself as for the "update" key.